core/domain/suggestion_registry.py

```diff
@@ -319,30 +318,124 @@ def accept(self, commit_message):
             self.final_reviewer_id, self.target_id, change_list,
             commit_message, is_suggestion=True)
 
-    @classmethod
-    def from_dict(cls, suggestion_dict):
-        """Return a Suggestion object of type SUGGESTION_TYPE_EDIT_STATE_CONTENT
-        from a dict.
 
-        Args:
-            suggestion_dict: dict. The dict representation of the suggestion.
+class SuggestionAddQuestion(BaseSuggestion):
+    """Domain object for a suggestion of type SUGGESTION_TYPE_ADD_QUESTION."""
 
-        Returns:
-            SuggestionEditStateContent. The corresponding Suggestion domain
-                object.
+    def __init__( # pylint: disable=super-init-not-called
+            self, suggestion_id, target_id, target_version_at_submission,
+            status, author_id, final_reviewer_id,
+            change_cmd, score_category, last_updated):
+        """Initializes an object of type SuggestionAddQuestion
+        corresponding to the SUGGESTION_TYPE_ADD_QUESTION choice.
```

Comment:
Add docstrings for args

Reply:
The docstrings for these args are in the BaseSuggestion class. Should I duplicate it here.

Reply:
Yes, because they take on different meanings in the subclasses. E.g. "target_id" for this one is the ID of the topic.